MacOSBuild.md

```diff
@@ -0,0 +1,18 @@
+How to build NIX-MX on MacOS
+----------------------------
+
+To compile NIX-MX for MaOS for your MATLAB Version follow these steps:
+
+**Dependencies**
+- you have to have a working build environment including a c/c++ compiler
```

Comment:
Start with captial y? Rmove the "c/" part before compiler. (I would also capitalize C++).